### PR TITLE
Add singlepage for confidential contact person

### LINKED ISSUE #2143: Add social security page or section to the site
### Is your feature request related to a problem? Please describe.

Because of the new Trust Contact Person project, we have to add some kind of social security page or section to our site, which students can easily find. A trust contact person is a computing science student that can help other students in finding the right help for them because that might sometimes be difficult to find or determine. The idea for now is that there are 2 of such Trust Contact Persons that will stay a Trust Contact Person for 2 years. They will not be replaced at the same time, but rather alternating per year, so that every year there is someone who already has experience with this and someone that is new. This means that it has to be changed once every year if everything goes as planned. Besides information about the trust contact person, we also want the other services of the radboud to be there, so students can easily find them.

### Describe the solution you'd like

We suggest to make a separate new page which contains:

- contact information of the Trust Contact Person. Note that every computing science student, not only Thalia members, must be able to contact this Trust Contact Person, because of that we suggest the following structure:
	* for non-members an email-address (and maybe a picture) would suffice
	* for members we can decide to also include mobile numbers if the Trust Contact Person approves
- links and/or contact information to all services the university offers (https://www.ru.nl/currentstudents/during-your-studies/overview-counsellors-advisors/student-support/). It is probably best to add a short description for every service so students have an idea what these persons are for. We can look for example at the sites of Marie-Curie (https://marie-curie.nl/page/show/93), Synergy (https://www.synergy.nu/en/members/mental-health)or Mycelium (https://mycelium.nu/studentbegeleiders/) for inspiration. 

We can add this for example as a drop-down menu at contact, or add a completely new section.

### Motivation
-

### Describe alternatives you've considered
-

### Additional context
-

### PR DESCRIPTION
**Pull request by JobDoesburg**
Closes #2143 

### Summary
Adds a HTML page on request of the board. It wasn't made into a dynamic app, as it is yet unclear how often the new idea will change.

### How to test
Visit the page

### CODE REVIEW

**Comment by JobDoesburg** [comment]
Waiting for the board

**Comment by JobDoesburg** [comment]
image
image

**Review by pingiun — DISMISSED** [review]

**Comment by JobDoesburg** [comment]
Let's wait a bit with merging. The board still wants to approve and they might want to wait with publishing it until somewhere in April when the contact persons have had their training. And I'd rather not cherry pick drop this commit in inbetween releases (especially considering the bootstrap5 update that's coming up)

**Review by se-bastiaan — CHANGES_REQUESTED** [review]
The resolutions of those images are _way_ too large. 7,8 MB for a single image? Dude.

Also please use some more descriptive file names.

File: website/singlepages/templates/singlepages/trust_contact_persons.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                person is confidential.
+            {% endblocktrans %}
+            <div class="row mt-4">
+                <div class="col-6 my-3">
+                    {% static "img/contact_persons/tom-koning.jpeg" as photo %}
+                    {% grid_item title="Tom Koning" meta_text="<p>gtkoning.tk@gmail.com</p>" url="mailto:gtkoning.tk@gmail.com" image_url=photo class_name="ratio2-3" %}
+                </div>
+                <div class="col-6 my-3">
+                    {% static "img/contact_persons/isolde-hoogendonk.jpg" as photo %}
+                    {% grid_item title="Isolde Hoogendonk" meta_text="<p>isoldeedlosi@gmail.com</p>" url="mailto:isoldeedlosi@gmail.com" image_url=photo class_name="ratio2-3" %}
```
Comment (se-bastiaan) [inline website/singlepages/templates/singlepages/trust_contact_persons.html:25]: Are we putting these email addresses in our code? Not a setting or something?

File: website/thaliawebsite/menus.py [hunk 2]
```diff
@@ -28,6 +28,7 @@
             },
             {"title": _("Become a Member"), "name": "registrations:index"},
             {"title": _("Alumni"), "name": "events:alumni"},
+            {"title": _("Trust contacts"), "name": "singlepages:trust-contacts",},
```
Comment (se-bastiaan) [inline website/thaliawebsite/menus.py:31]: Would 'student well-being' not be a better term for the whole thing?

File: website/thaliawebsite/static/css/base.scss [hunk 3]
```diff
@@ -332,14 +332,15 @@ img {
         height: 100%;
 
         display: flex;
-        align-items: center;
+        justify-content: flex-start;
+        align-items: stretch;
+        align-content: stretch;
```
Comment (se-bastiaan) [inline website/thaliawebsite/static/css/base.scss:337]: How does this affect all other grid items with images?

Comment (se-bastiaan) [reply website/thaliawebsite/static/css/base.scss:337]: Screenshot 2022-01-23 at 12 35 03

Quick copy paste in my inspector on the partners page and I say this breaks stuff.

**Comment by se-bastiaan** [comment]
@pingiun shame.

File: website/thaliawebsite/menus.py (hunk 2 above)
Comment (JobDoesburg) [reply website/thaliawebsite/menus.py:31]: @SnevelRetlaw

File: website/singlepages/templates/singlepages/trust_contact_persons.html (hunk 1 above)
Comment (se-bastiaan) [reply website/singlepages/templates/singlepages/trust_contact_persons.html:25]: And don't we just want these images to be square like the rest of the website?

Comment (JobDoesburg) [reply website/singlepages/templates/singlepages/trust_contact_persons.html:25]: I figured to make it as 'dumb' as possible. Making it a setting would also be possible, but then we could also just make it a true app

File: website/thaliawebsite/static/css/base.scss (hunk 3 above)
Comment (JobDoesburg) [reply website/thaliawebsite/static/css/base.scss:337]: Too bad, I didn't think about the the partner page

Comment (se-bastiaan) [reply website/thaliawebsite/static/css/base.scss:337]: Thabloid and documents use a custom height grid-item too :)
Maybe take a look at that as well if you want to change this

File: website/singlepages/templates/singlepages/trust_contact_persons.html (hunk 1 above)
Comment (se-bastiaan) [reply website/singlepages/templates/singlepages/trust_contact_persons.html:25]: I think I just missed the ending discussion part about it maybe. We have a template tag that directly outputs stuff from the settings. Seems easy to do, but I get your point. We have hardcoded email addresses in more places anyways.

File: website/thaliawebsite/menus.py (hunk 2 above)
Comment (SnevelRetlaw) [reply website/thaliawebsite/menus.py]: yes, we wanted to call it something like "student help" or "student well-being", so you can change it to "student well-being"

Comment (JobDoesburg) [reply website/thaliawebsite/menus.py]: I put it back under "education" and did the rename

**Review by se-bastiaan — APPROVED** [review]
:)

**Comment by SnevelRetlaw** [comment]
yes you can merge it!

**Pull request merged**